src/ontology/mondo-edit.obo

```diff
@@ -504065,12 +504065,15 @@ subset: rare
 synonym: "intellectual developmental disorder, autosomal dominant 64" EXACT [OMIM:619188]
 synonym: "mental retardation, autosomal dominant 64" EXACT DEPRECATED [OMIM:619188]
 synonym: "MRD64" EXACT ABBREVIATION [OMIM:619188]
+synonym: "ZNF292-related neurodevelopmental disorder" EXACT [https://doi.org/10.1002/cns3.20054, https://www.simonssearchlight.org/gene-guide/znf292/, PMID:31723249]
+synonym: "ZNF292-related syndrome" EXACT [https://doi.org/10.1002/cns3.20054, https://www.simonssearchlight.org/gene-guide/znf292/, PMID:31723249]
```

Comment:
This synonym should be "broad", as a variation in ZFF292 could cause syndromes that are beyond developmental disorders.

Reply:
updated